### DISCUSSION TITLE
Implement Streak Freeze

### DISCUSSION THREAD

**Discussion by bobbyiliev** [dailydotdev/daily#1289] [Feature Requests]
Love the Dail.Dev streak implementation! 

It will be cool if there was a streak freeze feature, allowing users to occasionally miss a day or two without loosing their streak. Similar to how Duolingo does it.

Streak feezes could be earned by doing some tasks or reputation points or similar.

**Comment by nimrodkra** [comment]
We are building it as we speak, stay tuned :)

**Reply by bobbyiliev** [reply]
Nice! 👏

**Comment by sabdailydev** [comment]
Was launched several weeks back :-), closing the thread.

**Comment by bobbyiliev** [comment]
It has been implemented! Closing this.
